services/src/main/java/io/druid/cli/CliPeon.java

```diff
@@ -106,7 +106,7 @@
 @Command(
     name = "peon",
     description = "Runs a Peon, this is an individual forked \"task\" used as part of the indexing service. "
-                  + "This should rarely, if ever, be used directly."
+                  + "This should rarely, if ever, be used directly. See http://druid.io/docs/latest/Peons.html for a description"
```

Comment:
http://druid.io/docs/latest/design/peons.html is more up to date.

Reply:
done